src/handlebars/archive.handlebars

```diff
@@ -19,14 +19,17 @@
 
     <div id="loading"><img src="dist/assets/loading_dots.gif" width="40" height="40"></div>
 
+    <!-- SEE THE COMMENTED-OUT FUNCTIONS IN ARCHIVE.JS
```

Comment:
Then can we kill the commented out code?  OK, I see what you're doing, I think a better way to handle this is to save that functionality off in a branch or as a patch or even just have a wiki entry stating that if we need it back go look at commit X.  It's usually a bit frowned upon to clutter up the source with lots of commented out code..